L/Libxc/build_tarballs.jl

```diff
@@ -3,28 +3,31 @@
 using BinaryBuilder, Pkg
 
 name = "Libxc"
-version = v"4.3.4"
+version = v"5.0.0"
 
 # Collection of sources required to complete build
 sources = [
-    ArchiveSource("https://gitlab.com/libxc/libxc/-/archive/4.3.4/libxc-4.3.4.tar.gz",
-                  "2d5878dd69f0fb68c5e97f46426581eed2226d1d86e3080f9aa99af604c65647"),
+    ArchiveSource("https://gitlab.com/libxc/libxc/-/archive/5.0.0/libxc-5.0.0.tar.gz",
+                  "6b3be3cf6daf6b3eddf32d4077276eb9169531b42f98c2ca28ac85b9ea408493"),
 ]
 
 # Bash recipe for building across all platforms
-# Note: Autotools fully supported upstream, but Windows builds only work with CMake
+# Notes:
+#   - Autotools fully supported upstream, but Windows builds only work with CMake
+#   - 3rd and 4th derivatives (KXC, LXC) not built since gives a binary size of ~200MB
 script = raw"""
 cd $WORKSPACE/srcdir/libxc-*/
 
 if [[ "${target}" = *-mingw* ]]; then
     mkdir libxc_build
     cd libxc_build
     cmake -DCMAKE_INSTALL_PREFIX=$prefix -DCMAKE_TOOLCHAIN_FILE=${CMAKE_TARGET_TOOLCHAIN} \
-        -DCMAKE_BUILD_TYPE=Release -DENABLE_FORTRAN=OFF -DBUILD_SHARED_LIBS=ON \
-        -DENABLE_XHOST=OFF ..
+        -DCMAKE_BUILD_TYPE=Release -DENABLE_FORTRAN=OFF -DENABLE_XHOST=OFF -DBUILD_SHARED_LIBS=ON \
+        -DDISABLE_VXC=OFF -DDISABLE_FXC=OFF -DDISABLE_KXC=NO -DDISABLE_LXC=NO ..
```

Comment:
Does "disable = no" mean "enable"? :thinking:

Reply:
Ups, bad typo on my end. It should be =ON. But now you already merged. Shall I make another PR?

Reply:
Yeah, maybe it's better.

What would save a lot of space is removing the static archives from the tarballs.  It would be great if we can disable them directly from the CMake configuration, otherwise
```
rm ${prefix}/lib/*.a
```
at the end of the build script should do the trick